frontend/src/plugin/__snapshots__/pluginLib.snapshot

```diff
@@ -16083,12 +16083,12 @@
     "flattenClusterListItems": [Function],
     "formatDuration": [Function],
     "getCluster": [Function],
+    "getClusterGroup": [Function],
     "getClusterPrefixedPath": [Function],
     "getPercentStr": [Function],
     "getReadyReplicas": [Function],
     "getResourceMetrics": [Function],
     "getResourceStr": [Function],
-    "getSelectedClusters": [Function],
```

Comment:
Why this change? Should it be in another commit?

Reply:
moved this to the correct commit, where rename actually happened